jira-slack-server-integration/jira-slack-server-integration-plugin/src/main/java/com/atlassian/jira/plugins/slack/service/listener/SlackEventHandlerService.java

```diff
@@ -168,7 +166,7 @@ public boolean handleMessage(@Nonnull final SlackIncomingMessage message) {
         List<IssueReference> issueReferences = extractIssueKeys(message.getText(), message.getLinks());
         log.debug("Found {} issue references", issueReferences.size());
 
-        UnfurlIssueLinksTask unfurlIssueLinksTask = taskBuilder.newUnfurlIssueLinksTask();
+        final List<Pair<JiraCommandEvent, NotificationInfo>> unfurlNotificationInfos = new ArrayList<>();
```

Comment:
Wondering if this could be inside the UnfurlIssueLinksTask class? I see that the UnfurlIssueLinksTask class has `List<Pair<JiraCommandEvent, NotificationInfo>>` defined